frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -48,10 +48,16 @@ const AdminShoutouts: React.FC = () => {
   }, [earlyDate, lastDate, hide, updateShoutouts]);
 
   const ListTitle = (): JSX.Element => {
-    let title = 'All Shoutouts';
-    if (view === 'HIDDEN') title = 'Hidden Shoutouts';
-    if (view === 'PRESENT') title = '';
-    return <Header className={styles.formTitle} content={title}></Header>;
+    let title = `Showing ${displayShoutouts.length} of ${allShoutouts.length} shoutouts`;
```

Comment:
Maybe this was a design decision you agreed on in work session, but I don't know if putting the total number of shoutouts makes sense. This number will only grow and grow as time goes on (unless we clear these out of the DB each semester?). It just doesn't seem like a number that's very relevant.

Reply:
@oscarwang20 Thanks for adding this change. I think I should have made this design choice more open for discussion. Could you make a post in #idol-devs about this and see what other think?